### ISSUE TITLE
HTTP command output in terminal.

### ISSUE THREAD

**Issue by akrasuski1** [radare/radare2#3940]
Normally, when we create a HTTP server using `=h` command, GET requests to `/cmd/stuff` should contain radare2's stdout response to `stuff`. However, *sometimes*, the output of that command is put into the console, instead of sent as a response.
I can't create a reproducible scenario - it seems to happen randomly, maybe once in a several hundred requests. Here is an example of what I mean: http://pastebin.com/C4NTVr59. As you can see, one of the requests, `drj`, had its output printed to console.

**Comment by radare** [comment]
Yep some commands are constructed by multiple calls of commands. This requires a console buffer push or a rewrite of the specific function to use the api instead of commands internally.

I'll investigate on this. Please report other commands that behave like this if you find them

Thanks

**Comment by radare** [comment]
Would be good to have a reproducer that make it happen 100% of the times.
